app/src/main/java/com/tari/android/wallet/ui/widget/CustomFont.kt

```diff
@@ -0,0 +1,29 @@
+package com.tari.android.wallet.ui.widget
+
+import android.content.Context
+import android.graphics.Typeface
+import java.util.*
+
+enum class CustomFont(val fileName: String) {
```

Comment:
What is the license for these fonts?

Reply:
This piece of code & fonts I just included as an example of custom font use. We may delete after we replace them with the fonts we'll receive from the designer.

Reply:
I don't think we would use Proxima Nova (just used it to make the demo look better really), and Font Awesome offers a free tier I think.

Reply:
https://fontawesome.com/license/free license for FontAwesome 5. FontAwesome can be pretty useful, and here's a list of free fonts in version 5: https://fontawesome.com/icons?d=gallery&m=free

Reply:
I have also included FontAwesome 4 in the project, which has some other useful fonts, and that one is free to use for anything it seems: https://fontawesome.com/v4.7.0/license/

Reply:
Removing them all in favor of the app fonts from the iOS app.